doc/requirements.rst

```diff
@@ -124,31 +128,33 @@ Motor 1.2.5 and 1.3.1 add compatibility with Python 3.7, but at the cost of
 dropping Python 3.4.3 and older. Motor 2.2 dropped support for Pythons older
 than 3.5.2. Motor 2.5 deprecates support for Python 3.5.
 
-+-------------------------------------------------------------------------------------------+
-|                   Python Version                                                          |
-+=====================+=====+=====+=====+=====+=====+=======+=======+=====+=====+=====+=====+
-|                     | 2.5 | 2.6 | 2.7 | 3.3 | 3.4 | 3.5.0 | 3.5.2 | 3.6 | 3.7 | 3.8 | 3.9 |
-+---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+-----+-----+
-| Motor Version | 1.0 |**N**|  Y  |  Y  |  Y  |  Y  |  Y    |  Y    |  Y  |**N**|**N**|**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+-----+-----+
-|               | 1.1 |**N**|  Y  |  Y  |  Y  |  Y  |  Y    |  Y    |  Y  |**N**|**N**|**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+-----+-----+
-|               | 1.2 |**N**|**N**|  Y  |**N**|  Y  |**N**  |  Y    |  Y  |  Y  |**N**|**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+-----+-----+
-|               | 1.3 |**N**|**N**|  Y  |**N**|  Y  |**N**  |  Y    |  Y  |  Y  |**N**|**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+-----+-----+
-|               | 2.0 |**N**|**N**|  Y  |**N**|  Y  |**N**  |  Y    |  Y  |  Y  |**N**|**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+-----+-----+
-|               | 2.1 |**N**|**N**|  Y  |**N**|  Y  |**N**  |  Y    |  Y  |  Y  |  Y  |**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+-----+-----+
-|               | 2.2 |**N**|**N**|**N**|**N**|**N**|**N**  |  Y    |  Y  |  Y  |  Y  |**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+-----+-----+
-|               | 2.3 |**N**|**N**|**N**|**N**|**N**|**N**  |  Y    |  Y  |  Y  |  Y  |**N**|
-+---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+-----+-----+
-|               | 2.4 |**N**|**N**|**N**|**N**|**N**|**N**  |  Y    |  Y  |  Y  |  Y  |  Y  |
-+---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+-----+-----+
-|               | 2.5 |**N**|**N**|**N**|**N**|**N**|**N**  |  Y    |  Y  |  Y  |  Y  |  Y  |
-+---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+-----+-----+
++-------------------------------------------------------------------------------------+
```

Comment:
This table doesn't render correctly anymore: https://motor--161.org.readthedocs.build/en/161/requirements.html#motor-and-python

Reply:
Fixed